src/Service/Startup.cs

```diff
@@ -401,20 +401,18 @@ private void AddGraphQLService(IServiceCollection services, GraphQLRuntimeOption
                 {
                     if (error.Exception is DataApiBuilderException thrownException)
                     {
-                        error = error.RemoveException()
-                                .WithMessage(thrownException.Message)
-                                .WithCode($"{thrownException.SubStatusCode}");
-
-                        // If user error i.e. validation error or conflict error with datasource, then retain location/path
-                        if (!thrownException.StatusCode.IsClientError())
```

Comment:
NIT: get rid of isClientError extension method if no longer needed

Reply:
this was an oversight in resolution of the merge conflict. Thanks for pointing it out. I will add this back.